src/ReadOutlets.cpp

```diff
@@ -86,30 +86,32 @@ int readoutlets(char *outletsds,char *lyrname, int uselayername,int outletslyr,O
 
 	if(hRSOutlet!=NULL) {
 
-	const char* epsgAuthorityIdRaster;
-	const char* epsgAuthorityIdOutlet;
-	int pj_raster=OSRIsProjected(hSRSRaster); // find if projected or not
-	int pj_outlet=OSRIsProjected(hRSOutlet);
-	OSRAutoIdentifyEPSG(hSRSRaster); //identify EPSG code
-	OSRAutoIdentifyEPSG(hRSOutlet);
-	const char *sprs;
-	if(pj_raster==0) {sprs="GEOGCS";} else { sprs="PROJCS"; }
-	if (pj_raster==pj_outlet){
-		 epsgAuthorityIdRaster=OSRGetAuthorityCode(hSRSRaster,sprs);// get EPSG code.  TODO.  Make sure these functions do not fail if there is no EPSG code
-	     epsgAuthorityIdOutlet=OSRGetAuthorityCode(hRSOutlet,sprs);
-
-	     if(atoi(epsgAuthorityIdRaster)!=atoi( epsgAuthorityIdOutlet)){
-	        printf( "Warning: EPSG code of Outlet shapefile and Raster data are different.\n" );
-			// TODO - Print the WKT and EPSG code of each.  If no spatial reference information, print unknown
-			// TODO - Test how this works if spatial reference information is incomplete, and create at least one of the unit test functions with a shapefile without a .prj file, and one of the unit test functions a raster without a projection (eg an ASCII file)
-		 }
-	}
+		int pj_raster=OSRIsProjected(hSRSRaster); // find if projected or not
+		int pj_outlet=OSRIsProjected(hRSOutlet);
+		const char *sprs;
```

Comment:
Check what happens if raster does not have projection.  hSRSRaster may be null and an error may occur